server/src/api/modules/competitions/services/FetchCompetitionDetailsService.ts

```diff
@@ -1,10 +1,11 @@
-import { omit } from 'lodash';
 import { z } from 'zod';
 import prisma, { modifyPlayer, modifySnapshot } from '../../../../prisma';
+import { omit } from '../../../util/objects';
 import { getMetricValueKey, isComputedMetric, Metric } from '../../../../utils';
 import { NotFoundError } from '../../../errors';
 import { CompetitionDetails } from '../competition.types';
 import * as deltaUtils from '../../deltas/delta.utils';
+import { Snapshot } from '@prisma/client';
```

Comment:
import this from `../../../../prisma` (as PrismaSnapshot) like modifyPlayer, modifySnapshot, etc

Reply:
Autocomplete got me 😆